express-api/src/routes/agenciesRouter.ts

```diff
@@ -15,13 +16,13 @@ const router = express.Router();
 // Endpoints for Admin Agencies
 router
   .route(`/`)
-  .get(protectedRoute(), catchErrors(getAgencies))
-  .post(protectedRoute([Roles.ADMIN]), catchErrors(addAgency));
+  .get(protectedRoute(), activeUserCheck, catchErrors(getAgencies))
```

Comment:
This probably should not have it, since we need to enumerate the agencies on the access request page before the keycloak user gets a row in the user table.

Reply:
Ah, I thought that came from the lookup routes. Will change.